sdk/src/assertions/mod.rs

```diff
@@ -77,3 +77,8 @@ mod embedded_data;
 pub use embedded_data::EmbeddedData;
 
 pub mod region_of_interest;
+
+mod soft_binding;
+pub use soft_binding::{
+    SoftBinding, SoftBindingBlockMap, SoftBindingScopeMap, SoftBindingTimespanMap,
```

Comment:
Why are we adding "Map" to the end of all these structures? What is that intended to convey. In Rust the practice is to avoid redundant information in naming. We are not supposed to add Enum or Enum names or Error or error names. Why not just call these SoftBindingBlock, SoftBindingScope and SoftBindingTimespan?